lib/screens/v2/candidate_generator/elevator.ex

```diff
@@ -26,29 +29,11 @@ defmodule Screens.V2.CandidateGenerator.Elevator do
     |> Builder.build_template()
   end
 
-  def candidate_instances(
-        config,
-        now \\ DateTime.utc_now(),
-        elevator_closure_instances_fn \\ &ElevatorClosures.elevator_status_instances/3,
-        evergreen_content_instances_fn \\ &Evergreen.evergreen_content_instances/2
-      ) do
-    Enum.concat([
-      elevator_closure_instances_fn.(
-        config,
-        header_instance(config, now),
-        footer_instance(config)
-      ),
-      evergreen_content_instances_fn.(config, now)
-    ])
+  @impl true
+  def candidate_instances(config, now \\ DateTime.utc_now(), instance_fns \\ @instance_fns) do
+    instance_fns |> Enum.map(& &1.(config, now)) |> Enum.concat()
```

Comment:
I'm seeing an error happening on this line
`expected a function, got: {:elevator_status, &Screens.V2.CandidateGenerator.Elevator.Closures.elevator_status_instances/2}`

If I'm understanding correctly, it seems like this is happening because `instance_fns` is actually a list of tuples rather than just a list of functions.

Reply:
The perils of mocking... 😬

Reply:
extremely relatable lol